runtime/moonriver/src/asset_config.rs

```diff
@@ -158,7 +158,7 @@ impl pallet_asset_manager::AssetRegistrar<Runtime> for AssetRegistrar {
 			},
 		)
 		.get_dispatch_info()
-		.weight
+		.call_weight
```

Comment:
```suggestion
		.total_weight()
```

Reply:
Addressed in https://github.com/moonbeam-foundation/moonbeam/pull/3224/commits/1c358f65551c35bded613759a980b99d0e60b0b3